calcom/cal.com
# feat: Add Skiff to app store

## What does this PR do?



Now that Skiff Calendar works with Cal.com, this PR adds Skiff as a link app to the app store.

Fixes # (issue)



## Requirement/Documentation


  - If there is a requirement document, please, share it here.
  - If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?



  - Are there environment variables that should be set?

Simply run the app store locally.

  - What are the minimal test data to have?

None needed.

  - What is expected (happy path) to have (input and output)?

See image and video below.

  - Any other important info that could help to test that PR

No other info needed.

## Mandatory Tasks


## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
